website/includes/donor-contact/fa-body.html

```diff
@@ -1,63 +1,265 @@
+              <div class="inputWrapper">
+                <input aria-required="true" type="text" id="tfa_10" name="tfa_10" value="" title="{{ _('Email') }}" class="validate-email required">
+              </div>
+            </div>
+          </div>
+          <div class="htmlSection" id="tfa_182">
+            <div class="htmlContent" id="tfa_182-HTML">
+              {{ _('Thunderbird will only use your submitted information for purposes of communicating with you about your request. See our <a href="%(url)s">privacy policy</a> for further information.')|format(url='https://www.mozilla.org/privacy/websites/') }}
+            </div>
+          </div>
         </div>
-        <div class="oneField field-container-D    " id="tfa_95-D">
-          <label id="tfa_95-L" class="label preField " for="tfa_95">{{ _('Subject') }}</label>
-          <div class="inputWrapper"><input type="text" id="tfa_95" name="tfa_95" value="" title="Subject" class="" required>
+        <div id="tfa_203" class="section group" data-condition="`#tfa_197`">
+          <div class="htmlSection" id="tfa_206">
+            <div class="htmlContent" id="tfa_206-HTML">
+              {{ _('We are unable to offer help with Thunderbird technical issues through the Donor Help form.') }}
+              <br>
+              <div>
+                {{ _('Visit our technical <a href="%(url)s" target="_blank">support page</a>. Our knowledge base and active user community can help!')|format(url='https://support.mozilla.org/products/thunderbird') }}
+              </div>
+              <br/>
+              <div>
+                <b>
+                  <a href="https://support.mozilla.org/products/thunderbird" target="_blank">{{ _('Visit our technical support site to get more help') }}</a>
+                </b>
+              </div>
+            </div>
           </div>
         </div>
-        <div id="tfa_183" class="section inline group">
-          <div class="oneField field-container-D    " id="tfa_163-D">
-            <label id="tfa_163-L" class="label preField " for="tfa_163">{{ _('Description') }}</label>
-            <div class="inputWrapper"><textarea id="tfa_163" name="tfa_163" title="Description" class="" required></textarea>
+        <div id="tfa_204" class="section group" data-condition="`#tfa_198`">
+          <div class="htmlSection" id="tfa_207">
+            <div class="htmlContent" id="tfa_207-HTML"><span>
+              {{ _('<a href="%(url)s" target="_blank">Mozilla Connect</a> is a collaborative space for ideas, feedback, and discussions that will help shape future Thunderbird product releases.')|format(url='https://connect.mozilla.org/') }}
+              {{ _('<a href="%(url)s" target="_blank">Submit your idea</a> and be sure to tag Thunderbird so others can find and vote on your suggestion.')|format(url='https://connect.mozilla.org/t5/ideas/idb-p/ideas/label-name/thunderbird') }}
```

Comment:
Maybe add this url to settings.py so it can just be `url('connect.thunderbird') or whatever and connect.mozilla.org can just be `url('connect')`.